slither.md

```diff
@@ -0,0 +1,105 @@
+```
```

Comment:
No reason to include this here, can run slither on demand as needed, but also not useful to run on the libraries/false positive on the base64 encode

Reply:
See the commented out slither command in the CI pipeline, would be preferable to just get that working and have the output on each pipeline.